src/styles/common.css

```diff
@@ -1,364 +1,355 @@
+}
+
+select {
+  padding: calc(var(--d) * 2) 0;
+  border-radius: var(--radius);
+}
+
+dt,
+dd,
+th,
+td {
+  padding: var(--d);
+  text-align: left;
+}
+
+abbr {
+  text-decoration: underline dotted 1px;
+  text-underline-offset: calc(var(--d) * 1.8);
+}
+
+cite {
+  display: block;
+  margin: calc(var(--d) * 4) 0;
+}
+
+pre {
+  white-space: pre-line;
+  background-color: var(--box-colour);
+  padding: calc(var(--d) * 2);
+}
+
+/* Classes */
+
+.box {
+  flex: 1 1 calc(20% - calc(var(--d) * 2));
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  padding: var(--d);
+}
+
+.card {
+  aspect-ratio: 1.618;
+}
+
+.square {
+  aspect-ratio: 1;
+}
+
+.flex-row {
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;
+  flex-wrap: wrap;
+  gap: var(--d);
+}
+
+.skip {
+  position: absolute;
+  left: -10000px;
+  top: auto;
+  width: 1px;
+  height: 1px;
+  overflow: hidden;
+}
+
+.skip:focus {
+  position: static;
+  width: auto;
+  height: auto;
+}
+
+.centre {
+  text-align: center;
+}
+
+.home-header {
+  display: flex;
+  align-items: center;
+  margin: calc(var(--logo-height) / 5) 0;
+}
+
+.home-header-logo {
+  height: var(--logo-height);
+  margin-right: calc(var(--logo-height) / 4);
+}
+
+.home-header-brand {
+  font-size: calc(var(--logo-height) / 3);
+  font-weight: bold;
+  letter-spacing: 0.02em;
+}
+
+.home-header-tagline {
+  font-size: calc(var(--logo-height) / 4.5);
+}
+
+.byline {
+  --byline-logo-height: calc(var(--logo-height) * 0.66);
+
+  display: flex;
+  align-items: center;
+  margin: calc(var(--byline-logo-height) / 3) 0 0;
+}
+
+.byline-logo {
+  height: var(--byline-logo-height);
+  margin-right: calc(var(--byline-logo-height) / 5);
+}
+
+.byline-brand {
+  font-size: calc(var(--byline-logo-height) / 3);
+}
+
+.byline-tagline {
+  font-size: calc(var(--byline-logo-height) / 4);
+}
+
+.tagline {
+  display: flex;
+  padding: 0.1em 0;
+  font-size: 0.8em;
+
+  & + div {
+    font-size: 0.7em;
+  }
+}
+
+.copyright {
+  font-size: calc(var(--footer-height) / 6);
+  text-align: right;
+  padding-top: calc(var(--footer-height) / 10);
+  letter-spacing: 0.01em;
+  font-family: sans-serif;
+}
+
+.header-text {
+  display: flex;
+  flex-direction: column;
+  place-content: center;
+  font-size: 1em;
+}
+
+.header-logo {
+  height: var(--logo-height);
+  margin-right: 0.5em;
+}
+
+@media (width >= 480px) {
+  .header-logo,
+  .tagline-logo {
+    display: block;
+  }
+}
+
+.subtitle {
+  font-style: italic;
+}
+
+.is-fg {
+  color: var(--bg-colour);
+  background-color: var(--fg-colour);
+}
+
+.is-bg {
+  background-color: var(--bg-colour);
+}
+
+.is-border {
+  background-color: var(--border-colour);
+}
+
+/* IDs */
+#writing {
+  scroll-margin-top: var(--header-height);
+}
+
+/* Overrides */
+:is(*):not(h1, h2, h3, h4, h5, h6, footer, nav, header) {
+  font-size: var(--max-font-size);
+}
+
+cite *,
+code {
+  font-size: calc(var(--d) * 4);
+  line-height: 1.6;
```

Comment:
Can you add these two lines:

```
overflow-wrap: break-word;
hyphens: auto;
```

Non-breaking `code` is what's allowing page width to exceed device width on some pages